tools/phenomenal/ms/save_chromatogram.xml

```diff
@@ -1,14 +1,20 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <!--Proposed Tool Section: [Statistics]-->
-<tool id="save_chromatogram" name="save_chromatogram" version="1.1">
+<tool id="save_chromatogram" name="save_chromatogram" version="1.2">
 <!--
   <requirements>
-    <container type="docker">phnmnl/xcms</container>
+    <container type="docker">container-registry.phenomenal-h2020.eu/phnmnl/xcms</container>
   </requirements>
 -->
-  <description>Saves the chromatogram of a single mzML file.</description>
+  <description>Saves the chromatogram in CSV format of a single mzML file.</description>
   <command><![CDATA[
-    save_chromatogram.r $infile $outfile 2>&1
+if [[ "\$(head -n 2 $infile | tail -n 1 | grep mzML)" != "" ]];
+then filename="\$(head -n 40 $infile | grep "sample id" | sed -e "s/\"\>.*//" | sed -e "s/.*name\=\"//")";
+    xcmsfile="/tmp/\$filename.mzML";
+    cp $infile \$xcmsfile;
```

Comment:
Please do a soft link instead of copying the file, otherwise we "leak" disk space unnecessarily here. A move would be a bad idea as well since Galaxy relies on the original file location for other purposes.

Reply:
and don't create it on the tmp, but on the same working directory :-), so something like:
```
xcmsfile="file.mzML";
ln -s $infile \$xcmsfile;
```
the tmp is not necessarily on the same disk were the working directory is, which could lead to all sort of issues depending on the galaxy installation. In general, one should avoid getting out of the working directory specified by Galaxy.